test/specs/validate-address.spec.js

```diff
@@ -84,13 +99,19 @@ describe("validateAddress()", () => {
   it("Validates a multi-line address", async function () {
     const shipengine = new ShipEngine({ apiKey, baseURL });
 
-    const response = await shipengine.validateAddress({
+    const addressToValidate = {
       country: "US",
-      street: ["4 Jersey St", "2nd Floor", "validate-multiline-address"],
+      street: [
+        "4 JERSEY ST STE 200",
+        "2nd Floor",
+        "validate-multiline-address",
+      ],
```

Comment:
I did that in the assertNormalizedAddressMatchesOriginal:

```expect(normalizedAddress.street[0].toUpperCase()).to.equal(
      originalAddress.street[0].toUpperCase()
    );

    if (normalizedAddress.street[2] !== undefined) {
      expect(normalizedAddress.street[2]).to.equal(
        originalAddress.street[2].toUpperCase()
      );
    }

    if (normalizedAddress.street[3] !== undefined) {
      expect(normalizedAddress.street[3]).to.equal(
        originalAddress.street[3].toUpperCase()
      );
    }
```
Would you prefer I move that into the test case to make it more explicit?

Reply:
You can definitely leave the logic in `assertNormalizedAddressMatchesOriginal()` as well, since it's useful to do that for _every_ test.  But for this specific test, you need to test more than just what's covered by `assertNormalizedAddressMatchesOriginal()`.  You need to ensure that the `street` array contains _only_ the expected number of items, and that _all_ of them match exactly what you expect.  The `assertNormalizedAddressMatchesOriginal()` function doesn't do that.

Reply:
Gotcha! Thanks for the quick response!